engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkScheduledExecutionPerformer.scala

```diff
@@ -63,7 +63,9 @@ class FlinkScheduledExecutionPerformer(
   }
 
   override def provideInputConfigDuringExecutionJson(): Future[InputConfigDuringExecution] =
-    Future.successful(inputConfigDuringExecution)
+    Future {
+      getCurrentInputConfigDuringExecution()
```

Comment:
During reading this config is used a strange model extension which can be reloaded so we have to access it lazily.